core/storage/user/gae_models_test.py

```diff
@@ -1631,6 +2087,24 @@ def test_get_deletion_policy(self) -> None:
             user_models.StoryProgressModel.get_deletion_policy(),
             base_models.DELETION_POLICY.DELETE)
 
+    def test_get_model_association_to_user(self) -> None:
+        self.assertEqual(
+            user_models.StoryProgressModel.get_model_association_to_user(),
+            base_models.MODEL_ASSOCIATION_TO_USER.MULTIPLE_INSTANCES_PER_USER)
+
+    def test_get_export_policy(self) -> None:
+        self.assertEqual(
+            user_models.StoryProgressModel.get_export_policy(),
+        {
```

Comment:
This and the following lines should be indented by 4 because you're still in scope of the '(' on line 2096.